server/datastore/mysql/software.go

```diff
@@ -895,3 +921,81 @@ ORDER BY
 	}
 	return hosts, nil
 }
+
+func (ds *Datastore) InsertVulnerabilities(
+	ctx context.Context,
+	vulns []fleet.SoftwareVulnerability,
+	source fleet.VulnerabilitySource,
+) (int64, error) {
+	var totalCount int64
+
+	if len(vulns) == 0 {
+		return totalCount, nil
+	}
+
+	var records []interface{}
+	for _, vuln := range vulns {
+		records = append(records, goqu.Record{
+			"cpe_id": vuln.CPEID,
+			"cve":    vuln.CVE,
+			"source": source,
+		})
+	}
+
+	iStmt, _, err := dialect.
+		Insert("software_cve").
```

Comment:
Any reason why not do SQL straight up? We haven't been using goqu for data changes anywhere yet.

Reply:
Yeah, I think we should keep it consistent with the rest.